packages/blade/src/components/DatePicker/_decisions/decisions.md

```diff
@@ -0,0 +1,361 @@
+
+## Anatomy
+
+<img src="./datepicker-range-example.png" width="60%" alt="DatePicker Anatomy" />
+
+## Components
+
+- Calendar (internal)
+- DatePicker
+
+## Basic Usage
+
+```jsx
+<DatePicker
+  label="Pick date"
+  selectionType="single"
+  defaultValue={new Date()}
+  onChange={(date) => {
+    console.log('date selected', date);
+  }}
+/>
+```
+
+## API
+
+### DatePicker API
+
+```ts
+type InputProps = Pick<
+  BaseInputProps,
+  | 'labelPosition'
+  | 'name'
+  | 'validationState'
+  | 'errorText'
+  | 'successText'
+  | 'helpText'
+  | 'necessityIndicator'
+  | 'isRequired'
+  | 'isDisabled'
+  | 'onFocus'
+  | 'onBlur'
+  | 'size'
+  | 'autoFocus'
+>;
+
+type DatePickerProps = InputProps &
+  CalendarProps & {
+    /**
+     * Label for the input
+     * 
+     * If labelPosition is set to "left" on range picker,
+     * then we will take the `{ start }` label and render it on the left side of the input ignoring the `{ end }`
+     */
+    label: string | { start: string, end?: string };
+    isOpen?: boolean;
+    defaultIsOpen?: boolean;
+    onOpenChange?: ({ isOpen }: { isOpen: boolean }) => void;
+  };
+```
+
+### CalendarProps
+
+```ts
+type PickerType = 'date' | 'month' | 'year';
+type DateRange = [Date, Date];
+type Preset = {
+  /**
+   * Label for the preset
+   *
+   * @example 'Last 7 days'
+   */
+  label: string;
+  /**
+   * Value can be a range of dates
+   *
+   * @example with range selection
+   * (date) => [dayjs().subtract(7, 'day'), dayjs()]
+   */
+  value: (date: Date) => DateRange;
+};
+
+type CalendarProps = {
+  /**
+   * Sets the selection mode of the calendar
+   * @default 'single'
+   */
+  selectionType: 'single' | 'range';
+  /**
+   * Sets the picker type
+   * @default 'date'
+   */
+  picker: PickerType;
+  onPickerChange: (picker: PickerType) => void;
+
+  // Standard controlled/uncontrolled state props
+  value: Date | DateRange;
+  defaultValue: Date | DateRange;
+  onChange: (value: Date | DateRange) => void;
+
+  /**
+   * Defines presets for the date range picker
+   *
+   * @example with range selection
+   * [
+   *   { label: 'Last 7 days', value: (date) => [dayjs(date).subtract(7, 'day'), date] },
+   *   { label: 'Last month', value: (date) => [dayjs(date).subtract(1, 'month'), date] },
+   * ]
+   */
+  presets: Preset[];
+  /**
+   * Sets the first day of the week in the calendar.
+   *
+   * 0-6 (0=Sunday, 1=Monday, etc.)
+   * @default 1
+   */
+  firstDayOfWeek: number;
+  /**
+   * Sets the minimum date that can be selected.
+   */
+  minDate?: Date;
+  /**
+   * Sets the maximum date that can be selected.
+   */
+  maxDate?: Date;
+  /**
+   * Determines whether single date can be selected as range, applicable only when type="range"
+   * @default false
+   */
+  allowSingleDateInRange?: boolean;
+  /**
+   * Sets the locale for the calendar.
+   *
+   * @default 'en'
+   */
+  locale?: string;
```

Comment:
What are the things that will be affected by this?

Reply:
Weekday names, month names etc. 

Check: https://mantine.dev/dates/dates-provider/